src/test/modelinstance_test.cpp

```diff
@@ -563,7 +563,7 @@ TEST_F(TestLoadModel, CheckSavedModelHandling) {
     auto model_files = modelInstance.getModelFiles();
     ASSERT_FALSE(model_files.empty());
 #ifdef _WIN32
-    EXPECT_EQ(model_files.front(), directoryPath + "/test_saved_model/1\\");
+    EXPECT_EQ(model_files.front(), directoryPath + "/test_saved_model\\1\\");
```

Comment:
Add function so we could use sth like this without macro's?
```suggestion
    EXPECT_EQ(model_files.front(), os_agnostic_path(directoryPath + "/test_saved_model/1/"));
```

Reply:
Are those the only places where we encounter with the different separator handling of Win/Lin?

Reply:
This is a case by case result for different tests. Part of path is from config or hardcoded and the rest is generated by ovms. So the number of ifdef would be the same.